packages/service-worker-mock/models/Response.js

```diff
@@ -1,7 +1,16 @@
 // stubs https://developer.mozilla.org/en-US/docs/Web/API/Response
-class Response {
+const Body = require('./Body');
+const Blob = require('./Blob');
+
+class Response extends Body {
   constructor(body, init) {
-    this.body = body || '';
+    if (typeof body === 'object' && !(body instanceof Blob)) {
+      body = JSON.stringify(body);
+    } else if (typeof body !== 'object') {
```

Comment:
Can you explain this line? I believe it will break if `body` is a string.

Reply:
JSON.stringify() does not break with a string

Reply:
`typeof 'foo' === 'string'`, so it won't get into that if statement, instead it will be set to `undefined`. That's why I was curious about the `else if` statement. What was the reasoning behind the fallback to `undefined`?